help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -80,13 +84,19 @@ The [Requests limits and allocations](https://docs.microsoft.com/en-us/power-pla
 
 {style="table-layout:auto"}
 
+## Supported audiences {#supported-audiences}
```

Comment:
Please check with engineering whether this destination supports external audiences or not.

Reply:
It doesn't support external audiences as there is a dependency on MSD's contactID.